catalog/app/components/Code/Code.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react'
+import * as M from '@material-ui/core'
+
+const useCodeStyles = M.makeStyles((t) => ({
+  root: {
+    background: t.palette.grey[300],
+    borderRadius: '2px',
+    color: t.palette.text.primary,
+    fontFamily: (t.typography as $TSFixMe).monospace.fontFamily,
```

Comment:
I just not familiar with redefining libraries. Should be something like:

```ts
import * as styles from '@material-ui/styles'

declare module '@material-ui/styles' {
  Typography extends styles.Typography {
    monospace: TypographyStyle
  }
}
```

But I couldn't make it work.

Reply:
i think it should be just something like `interface Typography { extra props here }`, see `app/@types/mui.d.ts`, tho not a big deal